balancer-js/src/modules/joins/joins.module.ts

```diff
@@ -556,6 +564,18 @@ export class Join {
           isLastChainedCall && minAmountsOut ? minAmountsOut[j] : '0';
 
         switch (node.joinAction) {
+          case 'wrap':
```

Comment:
I think we mentioned before that we would remove any join related stuff because we are focusing on exits?

Reply:
You're right - I'll remove them over here